dev-node/isbinaryfile/isbinaryfile-4.0.8.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit node
+
+MY_P="isBinaryFile-v${PV}"
+DESCRIPTION="Detects if a file is binary in Node.js. Similar to Perl's -B."
+HOMEPAGE="https://github.com/gjtorikian/isBinaryFile https://www.npmjs.com/package/isbinaryfile"
+SRC_URI="https://github.com/gjtorikian/isBinaryFile/releases/tag/${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/${MY_P}"
+RDEPEND="node-types/types+jest"
```

Comment:
se è in `devDependencies` mettilo in `BDEPEND`

Reply:
Ah ok